test/unit/tasks/publish_missed_scheduled_editions_test.rb

```diff
@@ -0,0 +1,27 @@
+require "test_helper"
+require "rake"
+
+class EditionsPublishMissedScheduledEditionsTaskTest < ActiveSupport::TestCase
```

Comment:
What's the reasoning behind the extra `Editions` at the start of the class name here?

Reply:
Yes it was because of the namespace, but actually it will be more readable without it 👍